modules/code-deployment/build.gradle

```diff
@@ -26,8 +26,12 @@ dependencies {
     annotationProcessor project(':ignite-configuration-annotation-processor')
     annotationProcessor project(':ignite-network-annotation-processor')
     annotationProcessor libs.auto.service
+    annotationProcessor libs.micronaut.inject.annotation.processor
+
 
     implementation libs.auto.service.annotations
+    implementation libs.micronaut.http.core
+    implementation libs.micronaut.http.server.core
```

Comment:
Since you've moved the controller to `rest-api` I don't think we need these micronaut dependencies anymore.